### PR TITLE
HUD Tutorial Fix Typos

### PR DESCRIPTION
**Pull request by kingthorin**
Per emailed feedback.

### CODE REVIEW

File: src/main/zapHomeFiles/hudtutorial/en_GB/SiteAlerts.html [hunk 1]
```diff
@@ -15,8 +15,8 @@ <h1>Site Alerts</H1>
 	The alerts are grouped by risk in the same way as Page Alerts:
 	<ul>
 	<li><img src="site-alerts-high.png"> High Level Site Alerts
-	<li><img src="site-alerts-low.png"> Low  Level Site Alerts
 	<li><img src="site-alerts-medium.png"> Medium Level Site Alerts
+	<li><img src="site-alerts-low.png"> Low Level Site Alerts
```
Comment (kingthorin) [inline src/main/zapHomeFiles/hudtutorial/en_GB/SiteAlerts.html]: Should we remove all occurrences of "Site" here as with the Page list?

In both cases should the `li` have closing tags?

File: src/main/zapHomeFiles/hudtutorial/en_GB/AlertNotifications.task.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 	<h2>Task</h2>
-	Click the button then keep your eyes on the bottom right hand side of the page - 
+	Click the submit button then keep your eyes on the bottom right hand side of the page -
```
Comment (thc202) [inline src/main/zapHomeFiles/hudtutorial/en_GB/AlertNotifications.task.html]: Start with upper case? (to match the button label)

File: src/main/zapHomeFiles/hudtutorial/en_GB/PageAlerts.html [hunk 3]
```diff
@@ -11,15 +11,15 @@ <h1>Page Alerts</H1>
 	which by default are shown on the left hand side. This allows you to just focus on the potential issues on
 	your current page without being distracted by issues on other pages. 
 	<p>
-	Alerts have an associated risk, and all alerts of with the same level of risk are shown together:
+	Alerts have an associated risk, and all alerts with the same level of risk are shown together:
 	<ul>
-	<li><img src="page-alerts-high.png"> High Level Page Alerts
```
Comment (thc202) [inline src/main/zapHomeFiles/hudtutorial/en_GB/PageAlerts.html]: I think they were as is to match the label of the tools.

File: src/main/zapHomeFiles/hudtutorial/en_GB/SiteAlerts.html (hunk 1 above)
Comment (thc202) [reply src/main/zapHomeFiles/hudtutorial/en_GB/SiteAlerts.html]: For consistency, if we don't mind the differences in the labels.

Fine either way.

File: src/main/zapHomeFiles/hudtutorial/en_GB/PageAlerts.html (hunk 3 above)
Comment (kingthorin) [reply src/main/zapHomeFiles/hudtutorial/en_GB/PageAlerts.html]: I'm not set on it one way or another. I see the value in matching it, but I also see Dave's point that it's kind of redundant. 

@psiinon tie break it? :smirk:

Comment (psiinon) [reply src/main/zapHomeFiles/hudtutorial/en_GB/PageAlerts.html]: Should def match the UI, and that means I think we should have both "Page" and "Site" in all places. If we dont make those things clear people wont understand the difference...

Comment (kingthorin) [reply src/main/zapHomeFiles/hudtutorial/en_GB/PageAlerts.html]: Ok, I'll revert the "Page" removals then.

**Comment by kingthorin** [comment]
Should be good to go now,.

**Comment by thc202** [comment]
Thank you!

**Comment by psiinon** [comment]
Nice!

**Pull request merged**